modules/targets/generic-linux/gpu/setup/default.nix

```diff
@@ -0,0 +1,32 @@
+# SPDX-FileCopyrightText: 2025 Jure Varlec <jure@varlec.si>
+#
+# SPDX-License-Identifier: MIT
+
```

Comment:
Not familiar with this but we already have an MIT license and you'll be in the git blame..

Reply:
I've grown the habit of making my stuff [REUSE compliant](https://reuse.software/), and as this code originates from my own repo, it didn't occur to me to remove these notices when I transfered the files here. Shall I remove them?